ruby/rails/security/brakeman/check-simple-format.yaml

```diff
@@ -0,0 +1,45 @@
+rules:
+- id: check-simple-format 
+  mode: taint
+  pattern-sources:
+  - pattern-either:
+    - pattern: |
+        cookies[...]
+    - patterns:
+      - pattern: |
+          cookies. ... .$PROPERTY[...]
+      - metavariable-regex:
+          metavariable: $PROPERTY
+          regex: (?!signed|encrypted)
+    - pattern: |
+        params[...]
+    - pattern: |
+        request.env[...]
+  pattern-sanitizers:
+  - pattern-either:
+    - pattern: h(...)
+    - pattern: html_escape(...)
+    - pattern: html_escape_once(...)
+  pattern-sinks:
+  - pattern: simple_format(...)
+  message: >-
+    Discovered usage of the `simple_format` helper, which
+    is vulnerable to XSS in versions 4.0.0 & 4.0.1. This
```

Comment:
This should probably be an SCA rule

Reply:
should i move these rules to the SCA repo then? Not sure what our current distinction between what should be in SCA or brakeman is 🤔

Reply:
This is probably OK, it would be higher accuracy with a project-depends-on key but this is probably okay as an audit rule.

Without the rails version context though, this will be noisy as `simple_format` is not a problem unless you are using a vulnerable rails version.